src/main/java/core/basesyntax/service/Operation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.OperationWithFruit;
+import core.basesyntax.strategy.OperationHandler;
+
+public interface Operation {
```

Comment:
```suggestion
public interface OperationHandler {
```
зараз назва більше схожа на назву моделі